README.md

```diff
@@ -1,7 +1,36 @@
-# hubTemplate
-
-**This repository is in development.**
-
-This repository serves as a template for groups wishing to build and maintain a new modeling hub, in the style of the [US COVID-19 Forecast Hub](https://github.com/reichlab/covid19-forecast-hub), the [US Scenario Modeling Hub](https://github.com/midas-network/covid19-scenario-modeling-hub) or the [European COVID-19 Forecast Hub](https://github.com/covid19-forecast-hub-europe/covid19-forecast-hub-europe). Researchers from these teams have created the Consortium of Infectious Disease Modeling Hubs to provide a [recommended set of standards and documentation](https://hubdocs.readthedocs.io/en/latest/index.html) for building and maintaining collaborative modeling hubs.
-
-This repository may be cloned to start a new repository for a modeling hub.
\ No newline at end of file
+<mark style="background-color: #FFE331">**This README.md is from [Flusight-Forecast_Hub  README](https://github.com/cdcepi/FluSight-forecast-hub/blob/master/README.md).  Be sure to modify the text below to reflect your hub.**</mark>
+
+
+# FluSight 2023-2024

```

Comment:
I think in general this is too much FluSight-specific detail for a repository that is designed to be just a template.